components/goals/GoalItem.js

```diff
@@ -0,0 +1,64 @@
+import Card from '../ui/Card';
+import GoalDetailModal from './GoalDetailModal';
+import PropTypes from 'prop-types';
+import { Typography, Button, Divider } from '@mui/material';
+import { useState } from 'react';
+
+function GoalItem({ title, owner, isEmployer }) {
+  const [open, setOpen] = useState(false);
+
+  function handleDetails() {
```

Comment:
Though it is okay, but It is advisable to use arrow functions

Reply:
I will never do this ))

Reply:
deal!